### PR TITLE
Improve docs.rs handling

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -25,13 +25,13 @@ name = "gdk_pixbuf"
 v2_32 = ["gdk-pixbuf-sys/v2_32"]
 v2_36 = ["v2_32", "gdk-pixbuf-sys/v2_36"]
 v2_36_8 = ["v2_36", "gdk-pixbuf-sys/v2_36_8"]
-dox = ["glib/dox", "gdk-pixbuf-sys/dox"]
+dox = ["glib/dox", "gdk-pixbuf-sys/dox", "futures"]
```
Comment (sdroege) [inline Cargo.toml]: Maybe combine this with removing of the `futures` feature here?

**Comment by sdroege** [comment]
Your commit message doesn't tell us in any way how this is better :) Otherwise apart from the `futures` feature this looks good to me.

File: Cargo.toml (hunk 1 above)
Comment (sdroege) [reply Cargo.toml]: Oh brainfart. We remove the `subclass` feature, not the `futures` feature. Ignore this part :)

Comment (GuillaumeGomez) [reply Cargo.toml]: Ignored then. :)

**Comment by GuillaumeGomez** [comment]
By not providing useless features to the docs.rs specific stuff. :)

**Comment by GuillaumeGomez** [comment]
Also, we do it this way for the `futures` feature handling in other crates. It seems good to me as is. Do you have something else in mind?

**Comment by sdroege** [comment]
> By not providing useless features to the docs.rs specific stuff. :)

I know that, but someone who reads the commit log or even this PR wouldn't :)

> Also, we do it this way for the `futures` feature handling in other crates. It seems good to me as is. Do you have something else in mind?

Seems fine as is. Once we require 1.39 we can remove the feature and always enable it.

**Comment by GuillaumeGomez** [comment]
> Seems fine as is. Once we require 1.39 we can remove the feature and always enable it.

If we all agree on supported rust starting 1.39, then we should create another issue for this and link it into the "before next release".

**Comment by GuillaumeGomez** [comment]
@sdroege I opened https://github.com/gtk-rs/gtk/issues/890

**Pull request merged**